casepro/cases/models.py

```diff
@@ -153,7 +153,7 @@ class Case(models.Model):
 
     initial_message = models.OneToOneField(Message, null=True, related_name='initial_case')
 
-    summary = models.CharField(verbose_name=_("Summary"), max_length=255)
+    summary = models.TextField(verbose_name=_("Summary"))
```

Comment:
The point of the summary field is to provide a *short* description of the case which can be used when listing cases. If users are trying to do something else with it, then maybe we're missing something, but removing this limit is not the solution.